website/src/theme/SearchPage/index.js

```diff
@@ -0,0 +1,459 @@
+/* eslint-disable jsx-a11y/no-autofocus */
+import React, {useEffect, useReducer, useRef, useState} from 'react';
```

Comment:
could you remind me why we needed to add an index.js file here? I think you mentioned it during standup but I can't remember what it actually does

Reply:
In the [issue](https://github.com/ecadlabs/taquito/issues/2858) I have [commented](https://github.com/ecadlabs/taquito/issues/2858#issuecomment-1992270479): 

> It seems to be a [known issue](https://github.com/facebook/docusaurus/issues/5084) that the search page does not respect the DocSearch options, so while turning the contextualSearch off changes the behaviour of the search modal, it does not affect the search page.
> 
> I swizzled the search page. In a separate commit, I modified it to respect the option. After merging, and verifying that the search works properly, it might be fixed for the searchpage as well, because a re-index with `contextualSearch:off` will fix that, according to the linked issue.
> 
> In that case, we can do a separate PR to undo the swizzling of the SearchPage, which is a good idea, as it reduces the code we need to maintain.